<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KF_GUIClass</class>
 <widget class="QWidget" name="KF_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kalman Filter</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 48);</string>
  </property>
  <widget class="QGroupBox" name="configureBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241);</string>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>147</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="libraryLayout">
     <item>
      <widget class="QLabel" name="libraryLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(241, 241, 241);</string>
       </property>
       <property name="text">
        <string>Library</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="radioButtonLayout">
       <item>
        <widget class="QRadioButton" name="asmRB">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>Assembly</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cppRB">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>C++</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="usedThreadsLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(241, 241, 241);</string>
         </property>
         <property name="text">
          <string>Threads used:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="usedThreadsNumber">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 122, 204);</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="threadNumberSlider">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(255, 255, 0);
alternate-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 0);
border-color: rgb(255, 255, 0);</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="inputLayout">
    <item>
     <widget class="QLabel" name="inputFileLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);</string>
      </property>
      <property name="text">
       <string>Input file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputFileLineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(241, 241, 241);background-color: rgb(67, 67, 70);</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="executionBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241);</string>
   </property>
   <property name="title">
    <string>Execution</string>
   </property>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 241, 241);
background-color: rgb(67, 67, 70);</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 241, 241);background-color: rgb(67, 67, 70);</string>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="statusTextLabel">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="assemblyLabel">
       <property name="text">
        <string>Assembly</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="asmTimeLabel">
       <property name="text">
        <string>----</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="usLabel2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>us</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cppLabel">
       <property name="text">
        <string>C++</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="cppTimeLabel">
       <property name="text">
        <string>----</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="usLabel1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>us</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Execution times:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>1271</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="chartsGrid">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="KF_GUI.qrc">:/KF_GUI/Logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KF_GUI.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>browseSlot()</slot>
  <slot>exitSlot()</slot>
  <slot>returnedPressedSlot()</slot>
 </slots>
</ui>
